hnswlib/hnswalg.h

```diff
@@ -616,6 +617,20 @@ namespace hnswlib {
             return;
         }
 
+        std::vector<dist_t> getDataByLabel(labeltype label)
```

Comment:
Type dist_t is for distances (e.g. result of the distance function), while the data vectors may have different type, so probably, a separate template type for this method should be better.